docs/source/getting-started.mdx

```diff
@@ -153,6 +153,15 @@ const server = new ApolloServer({
   resolvers,
   csrfPrevention: true,
   cache: 'bounded',
+  /**
+   * What's up with this embed: true option?
+   * These are our recommended settings for using AS;
+   * they aren't the defaults in AS3 for backwards-compatibility reasons but
+   * will be the defaults in AS4.
+  **/
+  plugins: [
+    ApolloServerPluginLandingPageLocalDefault({ embed: true }),
+  ],
```

Comment:
@glasser's comment (IIUC) was to add this bit to all of our code examples which `new ApolloServer(...)`. So any example where we've added `csrfPrevention: true` and `cache: 'bounded'`, we'd also add this `plugins` snippet (or append this plugin to an existing plugins array if the example has one).